TypeError: "file" argument must be a non-empty string on AArch64

The version of ovirt-web-ui is 1.1.0, the version of ovirt-engine-yarn is 0.24.4-1 and the version of ovirt-engine-nodejs is 8.0.0. All of them is come from oVirt 4.1 repo. Build it on CentOS 7 aarch64 and  failed with following message:
```bash
$ node scripts/build.js 
Building with homepagePath: /ovirt-engine/web-ui/
Creating an optimized production build...
child_process.js:360
    throw new TypeError('"file" argument must be a non-empty string');
    ^

TypeError: "file" argument must be a non-empty string
    at normalizeSpawnArguments (child_process.js:360:11)
    at spawnSync (child_process.js:487:38)
    at Compiler.checkFlowStatus (/root/rpmbuild/BUILD/ovirt-web-ui-1.1.0/node_modules/flow-babel-webpack-plugin/index.js:72:13)
    at Compiler.applyPluginsAsync (/root/rpmbuild/BUILD/ovirt-web-ui-1.1.0/node_modules/tapable/lib/Tapable.js:71:13)
    at Compiler.run (/root/rpmbuild/BUILD/ovirt-web-ui-1.1.0/node_modules/webpack/lib/Compiler.js:167:7)
    at build (/root/rpmbuild/BUILD/ovirt-web-ui-1.1.0/scripts/build.js:101:19)
    at recursive (/root/rpmbuild/BUILD/ovirt-web-ui-1.1.0/scripts/build.js:61:3)
    at /root/rpmbuild/BUILD/ovirt-web-ui-1.1.0/node_modules/recursive-readdir/index.js:31:14
    at FSReqWrap.oncomplete (fs.js:135:15)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
make: *** [ovirt-web-ui] Error 1
```

--- a/config/webpack.config.commons.js
+++ b/config/webpack.config.commons.js
@@ -1,0 +1,22 @@
+var process = require('process')
+
+var FlowBabelWebpackPlugin = require('flow-babel-webpack-plugin');
+
+/**
+ * Disable Flow type checker in unsupported environments.
+ *
+ * The condition `environmentSupportedByFlowBin` comes from `flow-bin` npm package. It can't be reliably reused because
+ * of poor design of FlowBabelWebpackPlugin that has `flow-bin` as its own dependency.
+ *
+ * @param webpackConfig webpack configuration object
+ */
+exports.addFlowBabelWebpackPlugin = function (webpackConfig) {
+  const environmentSupportedByFlowBin = process.platform === 'darwin'
+    || (process.platform === 'linux' && process.arch === 'x64')
+    || (process.platform === 'win32' &&  process.arch === 'x64')
+  if (environmentSupportedByFlowBin) {
+    webpackConfig.plugins.push(new FlowBabelWebpackPlugin())
+    return
+  }
+  console.warn(`Current environment (platform=${process.platform}, architecture=${process.arch}) is not supported by Flow type checker. Flow types will not be checked.`)
+}

--- a/config/webpack.config.dev.js
+++ b/config/webpack.config.dev.js
@@ -7,7 +7,7 @@ var WatchMissingNodeModulesPlugin = require('../scripts/utils/WatchMissingNodeMo
 var CopyWebpackPlugin = require('copy-webpack-plugin')
 var paths = require('./paths')
 var env = require('./env')
-var FlowBabelWebpackPlugin = require('flow-babel-webpack-plugin');
+var webpackConfigCommons = require('./webpack.config.commons')
 
 // This is the development configuration.
 // It is focused on developer experience and fast rebuilds.
@@ -223,7 +223,6 @@ module.exports = {
     // See https://github.com/facebookincubator/create-react-app/issues/186
     new WatchMissingNodeModulesPlugin(paths.appNodeModules),
 
-    new FlowBabelWebpackPlugin(),
     //global jquery is provided to any webpack modules
     /*new webpack.ProvidePlugin({
       $: "jquery",
@@ -239,3 +238,5 @@ module.exports = {
     tls: 'empty',
   },
 }
+
+webpackConfigCommons.addFlowBabelWebpackPlugin(module.exports)

--- a/config/webpack.config.prod.js
+++ b/config/webpack.config.prod.js
@@ -7,7 +7,7 @@ var CopyWebpackPlugin = require('copy-webpack-plugin')
 var url = require('url')
 var paths = require('./paths')
 var env = require('./env')
-var FlowBabelWebpackPlugin = require('flow-babel-webpack-plugin');
+var webpackConfigCommons = require('./webpack.config.commons')
 
 // Assert this just to be safe.
 // Development builds of React are slow and not intended for production.
@@ -245,8 +245,6 @@ module.exports = {
     }),
     // Note: this won't work without ExtractTextPlugin.extract(..) in `loaders`.
     new ExtractTextPlugin('static/css/[name].[contenthash:8].css'),
-
-    new FlowBabelWebpackPlugin(),
   ],
   // Some libraries import Node modules but don't use them in the browser.
   // Tell Webpack to provide empty mocks for them so importing them works.
@@ -256,3 +254,5 @@ module.exports = {
     tls: 'empty',
   },
 }
+
+webpackConfigCommons.addFlowBabelWebpackPlugin(module.exports)

